src/native/third_party/snappy.gyp

```diff
@@ -1,6 +1,15 @@
 # Copyright (C) 2016 NooBaa
 {
     'includes': ['common_third_party.gypi'],
+    'target_defaults': {
+        'conditions': [
+            ['node_arch=="s390x"', {
+                'conditions': [
+                    ['OS=="linux"', {'cflags': ['-DSNAPPY_IS_BIG_ENDIAN']}],
```

Comment:
do you need this OS=linux condition?

Reply:
Not sure, should I replace it with:
`[{'cflags': ['-DSNAPPY_IS_BIG_ENDIAN']}],`

Reply:
no, the total change will be:
```python
    'target_defaults': {
        'conditions': [
            ['node_arch=="s390x" or node_arch=="s390', { 'cflags': ['-DSNAPPY_IS_BIG_ENDIAN'] }],
        ],
    },
```